.github/workflows/lint.yaml

```
@@ -3,7 +3,8 @@ name: Scan
 on:
   # This workflow is triggered on pull requests to the main branch.
   pull_request:
-
+    # milestoned is added here as a workaround for release-please not triggering PR workflows (PRs should be added to a milestone to trigger the workflow).
+    types: [milestoned, opened, reopened, synchronize]
```

```suggestion
    types: [milestoned, opened, synchronize]
```
The other lints only had this but maybe we want reopened :shrug: